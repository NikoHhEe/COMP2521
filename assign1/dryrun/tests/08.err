==6341== Memcheck, a memory error detector
==6341== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==6341== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==6341== Command: ./testTextbuffer 08
==6341== 
==6341== 
==6341== Process terminating with default action of signal 6 (SIGABRT)
==6341==    at 0x4A3E85B: raise (raise.c:51)
==6341==    by 0x4A29534: abort (abort.c:79)
==6341==    by 0x10B2FE: range (textbuffer.c:270)
==6341==    by 0x10B619: deleteTB (textbuffer.c:373)
==6341==    by 0x10B485: copyTB (textbuffer.c:317)
==6341==    by 0x10B51A: cutTB (textbuffer.c:342)
==6341==    by 0x10A352: doCutTB (testTextbuffer.c:349)
==6341==    by 0x1098C6: test08 (testTextbuffer.c:151)
==6341==    by 0x1093F7: main (testTextbuffer.c:72)
==6341== 
==6341== HEAP SUMMARY:
==6341==     in use at exit: 421 bytes in 27 blocks
==6341==   total heap usage: 50 allocs, 23 frees, 17,647 bytes allocated
==6341== 
==6341== LEAK SUMMARY:
==6341==    definitely lost: 0 bytes in 0 blocks
==6341==    indirectly lost: 0 bytes in 0 blocks
==6341==      possibly lost: 0 bytes in 0 blocks
==6341==    still reachable: 421 bytes in 27 blocks
==6341==         suppressed: 0 bytes in 0 blocks
==6341== Rerun with --leak-check=full to see details of leaked memory
==6341== 
==6341== For counts of detected and suppressed errors, rerun with: -v
==6341== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
