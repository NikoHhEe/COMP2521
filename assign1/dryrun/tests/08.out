Test 08 - cutTB - cutting from the middle
Read in TB from 'simple-1.txt' into tb1
> linesTB(tb1);
7
> dumpTB(tb1, false);
line 01
line 02
line 03
line 04
line 05
line 06
line 07
> tb2 = cutTB(tb1, 3, 6);
Out of Range
